bucket/nicotine-plus.json

```diff
@@ -0,0 +1,51 @@
+{
+    "version": "3.1.1",
+    "description": "Graphical client for the Soulseek file sharing network",
+    "homepage": "https://nicotine-plus.org/",
+    "license": {
+        "identifier": "GPL-3.0-or-later",
+        "url": "https://github.com/nicotine-plus/nicotine-plus/blob/master/COPYING"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/nicotine-plus/nicotine-plus/releases/download/3.1.1/windows-x86_64-package.zip",
+            "hash": "3400d2273840ac2b1a69d9f2c89df15c0f4830c69b0f658a8a67f5071c22a10c"
+        },
+        "32bit": {
+            "url": "https://github.com/nicotine-plus/nicotine-plus/releases/download/3.1.1/windows-i686-package.zip",
+            "hash": "cb6c059cc36ce73b23e093929c0846c262b1d3fd62e3d76849567363d2a4064c"
+        }
+    },
+    "extract_dir": "Nicotine+",
+    "bin": [
+        [
+            "Nicotine+.exe",
+            "Nicotine+"
+        ]
+    ],
```

Comment:
Why is the useless rewrite needed??

Reply:
I haven't written one of these manifest before, I was simply following some examples. I will make the changes.

Reply:
I assume it can just be:

```json
"bin": [
    [
        "Nicotine+.exe"
    ]
],
```

Correct?